Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Search/OneWay.hs

```diff
@@ -86,7 +92,9 @@ oneWaySearch ::
 oneWaySearch personId req bundleVersion clientVersion device = do
   person <- QP.findById personId >>= fromMaybeM (PersonDoesNotExist personId.getId)
   merchant <- QMerc.findById person.merchantId >>= fromMaybeM (MerchantNotFound person.merchantId.getId)
-
+  mbFavourite <- findByLatLon req.origin.gps
```

Comment:
can we expect this also from frontend ? frontend already has this info, we can save a DB call here.

Reply:
that this location is favourite or not?

Reply:
yes